cmd/vela-worker/exec.go

```diff
@@ -19,6 +19,7 @@ import (
 
 // exec is a helper function to poll the queue
 // and execute Vela pipelines for the Worker.
+// nolint:funlen
```

Comment:
Could we add a description for the `nolint` comment here?

Perhaps something like:

```suggestion
// nolint:funlen // ignore function length due to comments
```

Reply:
We can add something, but I don't think it's the comments that are making the function too long since the function is over 100 lines with comments and the threshold is at 50. Do you have any suggestions?

Reply:
FYI:

I think the threshold for lines is actually set at `100` based off our configuration:

https://github.com/go-vela/worker/blob/45391b6040590d19c779da539e5f0e7acd8fab73/.golangci.yml#L29-L32

The linter source code can be found here:

https://github.com/ultraware/funlen

It measures both lines and statements and I think the issue here was about the statements [based off the Actions output](https://github.com/go-vela/worker/runs/1663678947):

![image](https://user-images.githubusercontent.com/10966923/104061535-654b4580-51be-11eb-8ac3-851c383eb970.png)

All that being said, I can see what you're talking about maybe it's not only comments.

Perhaps we also blame log messages as well? 😅 

```suggestion
// nolint:funlen // ignore function length due to comments and log messages
```

Reply:
I think that's a fair comment 😄 

Thank you for digging into that and providing more info than I did. I'm going to steal that comment for now!